test/smtcallback.js

```diff
@@ -72,10 +72,20 @@ tape('SMTCheckerCallback', function (t) {
       return { error: 'Fake SMT solver error.' };
     };
 
+    var pragmaSMT = '';
+    var settings = {};
+    // `pragma experimental SMTChecker;` was deprecated in 0.8.4
+    if (!semver.gt(solc.semver(), '0.8.3')) {
```

Comment:
Perhaps `lte` for better readability instead of not greater than?

Reply:
the problem with that is pre-releases, so `< 0.8.4` actually wouldn't work for the solidity PR itself. That's why here and in other places we do this instead